docs/nidcpower/class.rst

```diff
@@ -135,7 +135,7 @@ abort
             when you call the :py:meth:`nidcpower.Session.abort` method, the output channels remain
             in their current state and continue providing power.
 
-            Use the :py:meth:`nidcpower.Session.ConfigureOutputEnabled` method to disable power
+            Use the :py:meth:`nidcpower.Session.configure_output_enabled` method to disable power
```

Comment:
These changes shouldn't be shown in this PR.

Reply:
I have redone this as a new PR to get rid of the extraneous changes. See #1112